### PR TITLE
Disable BuildIntegratedLockFileIsCreatedOnBuild as it's a flaky test

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/1879
 
Regression? Last working version:

## Description

Given that https://github.com/NuGet/NuGet.Client/pull/4821 is still in draft mode, and this is blocking merges due to failures, I think we should disable this test until https://github.com/NuGet/Home/issues/12104 is fixed. 

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [x] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [x] N/A

### CODE REVIEW

**Review by jeffkl — DISMISSED** [review]

File: test/EndToEnd/tests/BuildIntegratedTest.ps1
Comment (zivkan) [inline test/EndToEnd/tests/BuildIntegratedTest.ps1:114]: I don't think this attibute works without a `param()` line below it. See other (working) examples of skipped e2e tests.

**Review by jeffkl — APPROVED** [review]

File: test/EndToEnd/tests/BuildIntegratedTest.ps1
Comment (nkolev92) [reply test/EndToEnd/tests/BuildIntegratedTest.ps1:114]: Other tests seem to have the param. Rather than waiting for the build to pass or testing manually (which would take long), I'll trust you :D

**Review by zivkan — APPROVED** [review]

**Comment by kartheekp-ms** [comment]
@nkolev92 - please don't merge this PR. Tests running on 17.4 fail 100% of times because of https://developercommunity.visualstudio.com/t/VS-2022-174-preview-versions-throws-a/10157815 problem.

**Comment by nkolev92** [comment]
> @nkolev92 - please don't merge this PR. Tests running on 17.4 fail 100% of times because of https://developercommunity.visualstudio.com/t/VS-2022-174-preview-versions-throws-a/10157815 problem.

I'm not sure I understand this. 
I just want to unblock dev for merging PRs. 
Test failures should not affect our code velocity and they are doing that right now.

**Comment by nkolev92** [comment]
We synced up and @kartheekp-ms and @dominoFire suggest that the moment one test is fixed, the next one will fail. The reason for that is because the test failure causes a modal dialog to popup and there's no one that can close it for the next test to be run.

tldr; This PR won't succeed.

**Pull request closed without merging**
